docs/stacks-guide-azure.md

```diff
@@ -0,0 +1,426 @@
+---
+title: "Crossplane Stacks Guide: Azure Setup" toc: true weight: 332 indent: true
```

Comment:
The `Rewrap` seems to have joined these lines

Reply:
Good catch! Thanks :)